app/Models/Chat/Channel.php

```diff
@@ -389,10 +393,22 @@ public function isValid()
 
         if ($this->name === null) {
             $this->validationErrors()->add('name', 'required');
+        } else if (mb_strlen($this->name) > static::MAX_LENGTHS['name']) {
```

Comment:
can use similar logic with existing `MAX_FIELD_LENGTHS` in other models. May even refactor the check to be shared between models.

Reply:
so that's the name I was looking for :dernk: